chart/values.yaml

```diff
@@ -5141,6 +5141,8 @@ externalProxy:
   sslRejectUnauthorized: true
   ## @param externalProxy.sslCA CA for the proxy SSL certificate in case is self-signed or signed by a not well-known CA
   sslCA: ""
+  ## @param externalProxy.sslCAConfigmapName Name of the ConfigMap containing the CA certificate for the proxy SSL certificate
+  sslCAConfigmapName: ""
```

Comment:
You should also define the key of the configmap, right?

Also, is that our standard way when we allow pass the value/secret or reference it from an external place? (like pass the raw secrets value vs secretName+key)

Please, keep thinking first on helm and then on Kots, in that way we can create Helm package usable without breaking replicated things.

Reply:
Ah, much better! Yes it totally makes sense, I'm checking with replicated team how to handle the key that is used in the secret. I'll update it ASAP, thanks!

My initial idea was to force customers to create a secret identical to the one that Replicated creates to avoid issues but it's much easier if we allow defining the key and we hardcode it when using KOTS